docs/en/action-menu-items/custom-browser-protocols.md

```diff
@@ -162,9 +162,9 @@ By varying the keyword after the `//` part of the URL in your AMI, you can chang
 
 Using this methodology you could open applications, upload content via services like FTP, archive data, send email, or generate PDF reports.
 
-## Registering a protocol on OSX
+## Registering a protocol on OSX (BigSur and Monterey)
```

Comment:
OSX is an outdated name -> https://en.wikipedia.org/wiki/MacOS_version_history#Releases
We should use macOS instead. Especially if we reference such recent versions.

Reply:
What about Ventura? Do we support it yet? Is this section applicable?

Reply:
Thanks @julien-lang . Not sure about Ventura. Who can we ask?

Reply:
Not sure too but it's a good guess that Ventura applies too. Basically, as far as we know, BigSur and newer (version 11+).

Please make sure you ask QA to validate your changes so we can test with Ventura.